tests/functional/generate_pages.py

```diff
@@ -440,25 +427,62 @@ def process_final_summary(
             append_spec_page_import(block_context, spec_file)
 
 
+def process_view_submitted_response(schema_data, require_path, dir_out, spec_file):
+    page_filename = f"view-submitted-response.page.js"
+    page_path = os.path.join(dir_out, page_filename)
+
+    logger.info("creating %s...", page_path)
+
+    with open(page_path, "w") as page_spec:
+        block_context = build_and_get_base_page_context(
+            page_dir=dir_out.split("/")[-1],
+            page_spec=page_spec,
+            base_page="BasePage",
+            base_page_file="base.page.js",
+            page_name="ViewSubmittedResponse",
+            page_filename=page_filename,
+            page_id="view-submitted-response",
+            relative_require=require_path,
+        )
+
+        for section in schema_data["sections"]:
+            write_summary_spec(
+                page_spec,
+                section,
+                collapsible=False,
+                summary_type="view_submitted_response",
+            )
```

Comment:
Ideally this should only write out things relevant to view submission, but this also contains things like edits links which are redundant.